src/pages/docs/guides/chat/build-livestream.mdx

```diff
@@ -0,0 +1,240 @@
+
+When implementing chat moderation, consider these essential factors:
+
+1. **Community standards**
+   - What level of moderation is appropriate for your audience?
+   - How will you handle different types of content? For example:
+     - **Hate speech and harassment:** Detecting discriminatory language, threats, or targeted abuse
+     - **Inappropriate content:** Flagging adult content, violence, or graphic material
+     - **Toxicity:** Measuring overall message sentiment and hostility
+   - AI moderation systems typically classify text according to categories similar to those above, usually with a severity and confidence level. If you have a provider in mind, what thresholds would you set for your chat?
+   - Do you need different rules for different user groups?
+
+2. **Technical requirements**
+   - How latency-sensitive is your chat?
+     - Inline AI moderation can add up to 100ms to message delivery
+   - Will you use a third-party moderation service or build your own?
+
+3. **Community-specific approaches**
+   - **Gaming streams:** More relaxed rules for casual language and humor
+   - **News and current affairs:** Strict moderation of misinformation and hate speech
+   - **Educational content:** Emphasis on respectful discussion and spam prevention
+
+### How Ably helps keep your chats safe
+
+Ably's integration with moderation providers allows you to customise the moderation policy on a per-chat basis, allowing you to implement different rules and sensitivity levels for different types of content and communities. You can even configure what happens if the moderation provider can't be reached - for example, publishing the message anyway. Get up and running with **out-of-the box integrations in minutes** and try out many providers for free. Ably takes care of all of the complex bits of invoking the moderation provider, handling the results and failure conditions.
+
+Of course, you may already have built moderation in other parts of your organisation and want to re-use it. Ably's got you covered! "Bring your own" moderation rules allow you to connect your chat rooms to your own infrastructure via popular transports including serverless functions, queues and webhooks. By implementing simple APIs, you can integrate your own moderation solution seamlessly with Ably.
+
+CODE HERE: Example of handling moderation failures in Ably Chat
+
+PLACEHOLDER: Diagram showing moderation flow (user → Ably → moderation → delivery)
+
+## Occupancy: Give volume to your livestream
+
+Ably's occupancy features provide you with realtime statistics of how many users are engaging with your live stream. These can be used to hype up your streams and draw in users to be part of the experience.
+
+- **Within a stream:** Show the number of active users in real time.
+- **Across streams:** Aggregate occupancy data for stream lists or dashboards.
+
+CODE HERE: Example of retrieving occupancy metrics
+
+## Reactions, typing indicators, and ephemeral events
```

Comment:
I'm not sure "ephemeral events" is something that would catch my eye, and they're essentially the same thing as reactions and indicators here.